src/pages/index.js

```diff
@@ -4,114 +4,283 @@ import FormValidator from "../components/FormValidator.js";
+
+function handleImageClick(card) {
+  previewImagePopup.open(card);
+}
+
+function createCard(cardData) {
+  const card = new Card(
+    cardData,
+    "#card-template",
+    handleImageClick,
+    handleDeleteClick,
+    handleAddLike
+  );
+  return card.getView();
+}
 
-/* Event Listeners */
+/* -------------------------------------------------------------------------- */
+/*                               Event Listeners                              */
+/* -------------------------------------------------------------------------- */
 
-profileEditButton.addEventListener("click", () => {
-  const { name, description } = userInfo.getUserInfo();
+avatarImage.addEventListener("click", () => {
+  avatarEditForm.open();
+});
+
+pencil.addEventListener("click", () => {
+  avatarEditForm.open();
+});
+
+addNewCardBtn.addEventListener("click", () => {
+  addCardFormValidator.resetValidation();
+  addImageForm.open();
+});
+
+profileEditBtn.addEventListener("click", () => {
+  const { name, description } = profileUserInfo.getUserInfo();
   profileTitleInput.value = name;
-  profileDescriptionInput.value = description;
-  editCardFormValidator.resetValidation();
-  profileEditPopup.open();
+  profileSubtitleInput.value = description;
+  profileEditForm.open();
+});
+
+modalImage.addEventListener("click", () => {
+  previewImagePopup.open();
+});
+
+/* -------------------------------------------------------------------------- */
+/*                                   Classes                                  */
+/* -------------------------------------------------------------------------- */
+
+const profileEditImage = new FormValidator(config, editImageForm);
+profileEditImage.enableValidation();
+
+const addCardFormValidator = new FormValidator(config, addCardForm);
+addCardFormValidator.enableValidation();
+
+const editProfileFormValidator = new FormValidator(config, profileForm);
+editProfileFormValidator.enableValidation();
+
+const profileEditForm = new PopupWithForm(
+  "#profile-edit-modal",
+  handleProfileEditSubmit
+);
+profileEditForm.setEventListeners();
+
+const profileUserInfo = new UserInfo({
+  title: ".profile__title",
+  subtitle: ".profile__subtitle",
+  avatar: ".profile__image",
 });
 
-const addCardFormValidator = new FormValidator(
-  formValidationSettings,
-  addCardForm
+const addImageForm = new PopupWithForm("#add-card-modal", handleCardSubmit);
+addImageForm.setEventListeners();
+
+const previewImagePopup = new PopupWithImage("#preview-modal");
+previewImagePopup.setEventListeners();
+
+const confirmDeletePopup = new PopupConfirm("#delete-popup");
+confirmDeletePopup.setEventListeners();
+
+const avatarEditForm = new PopupWithForm(
+  "#profile-image-modal",
+  handleAvatarSubmit
 );
+avatarEditForm.setEventListeners();
 
-const editCardFormValidator = new FormValidator(
-  formValidationSettings,
-  profileEditForm
+const avatarEditFormValidation = new FormValidator(
+  config,
+  profileEditImageForm
```

Comment:
You've created 2 `avatar` form validators: `profileEditImage` and `avatarEditFormValidation`. Please, delete one of them